qcfractal/interface/client.py

```diff
@@ -130,6 +131,59 @@ def _request(self,
 
         return r
 
+    def _automodel_request(self,
+                           name: str,
+                           rest: str,
+                           payload: Dict[str, Any],
+                           full_return: bool=False,
+                           timeout: int=None) -> Any:
+        """Automatic model request profiling and creation using rest_models
+
+        Parameters
+        ----------
+        name : str
+            The name of the REST endpoint
+        rest : str
+            The type of REST endpoint
+        payload : Dict[str, Any]
+            The input dictionary
+        full_return : bool, optional
+            Returns the full server response if True that contains additional metadata.
+        timeout : int, optional
+            Timeout time
+
+        Returns
+        -------
+        Any
+            The REST response object
+
+        Raises
+        ------
+        TypeError
+            Description
+
+        Deleted Parameters
+        ------------------
+        noraise : bool, optional
+            Optionally do not raise error if
+            Description
+        """
+        body_model, response_model = rest_model(name, rest)
+
+        # Provide a reasonable traceback
+        try:
+            payload = body_model(**payload)
+        except ValidationError as exc:
+            raise TypeError(str(exc))
```

Comment:
Why not just raise a `ValidationError` here?

Reply:
Print out complexity, we can reconsider later.